src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -160,11 +206,14 @@ public TransceiverCodec TransceiverCodec
         /// Retrieves the supported transceiver codecs.
         /// </summary>
         /// <remarks>
-        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.
+        /// This API is not supported in <see cref="MediaFileSource"/>, <see cref="MediaPacketSource"/>.<br/>
+        /// If user want to get supported codecs for each audio or video in<see cref="MediaNullSource"/>,
```

Comment:
please put a space between "video in" and `<see`

Reply:
Fixed